data/locale/jack_mixer-es.po

```diff
@@ -0,0 +1,832 @@
+
+#: jack_mixer/app.py:169
+msgid "jack_mixer XML files"
+msgstr "Archivos XML de jack_mixer"
+
+#: jack_mixer/app.py:182
+msgid "_Recent Projects"
+msgstr "_Proyectos Recientes"
+
+#: jack_mixer/app.py:218
+msgid "_Mixer"
+msgstr "_Mezclador"
+
+#: jack_mixer/app.py:220
+msgid "_Edit"
+msgstr "_Editar"
+
+#: jack_mixer/app.py:222
+msgid "_Help"
+msgstr "_Ayuda"
+
+#: jack_mixer/app.py:230
+msgid "New _Input Channel"
+msgstr "Nuevo Canal de _Entrada"
+
+#: jack_mixer/app.py:234
+msgid "New Output _Channel"
+msgstr "Nuevo _Canal de Salida"
+
+#: jack_mixer/app.py:241
+msgid "_Open..."
+msgstr "_Abrir..."
+
+#: jack_mixer/app.py:247
+msgid "_Save"
+msgstr "_Guardar"
+
+#: jack_mixer/app.py:251
+msgid "Save _As..."
+msgstr "Guardar _Como..."
+
+#: jack_mixer/app.py:256
+msgid "_Hide"
+msgstr "_Ocultar"
+
+#: jack_mixer/app.py:258
+msgid "_Quit"
+msgstr "_Salir"
+
+#: jack_mixer/app.py:265
+msgid "_Edit Input Channel"
+msgstr "_Editar Canal Entrada"
+
+#: jack_mixer/app.py:272
+msgid "E_dit Output Channel"
+msgstr "E_ditar Canal Salida"
+
+#: jack_mixer/app.py:279
+msgid "_Remove Input Channel"
+msgstr "E_liminar Canal Entrada"
+
+#: jack_mixer/app.py:286
+msgid "Re_move Output Channel"
+msgstr "Eli_minar Canal Salida"
+
+#: jack_mixer/app.py:294
+msgid "Shrink Channels"
+msgstr "Encoger Canales"
+
+#: jack_mixer/app.py:297
+msgid "Expand Channels"
+msgstr "Expandir Canales"
+
+#: jack_mixer/app.py:301
+msgid "_Clear"
+msgstr "_Va_ciar"
+
+#: jack_mixer/app.py:306
+msgid "_Preferences"
+msgstr "_Preferencias"
+
+#: jack_mixer/app.py:313
+msgid "_About"
+msgstr "_Acerca de"
+
+#: jack_mixer/app.py:363
+msgid "Input channel creation failed."
+msgstr "Error al crear canal de entrada."
+
+#: jack_mixer/app.py:425
+msgid "Output channel creation failed."
+msgstr "Error al crear canal de salida."
+
+#: jack_mixer/app.py:484 jack_mixer/app.py:573 jack_mixer/app.py:1158
+#, python-brace-format
+msgid "Error loading project file '{filename}': {msg}"
+msgstr "Error al cargar el archivo de proyecto '{filename}': {msg}"
+
+#: jack_mixer/app.py:558
+msgid "XML files"
+msgstr "Archivos XML"
+
+#: jack_mixer/app.py:562
+msgid "All files"
+msgstr "Todos los archivos"
+
+#: jack_mixer/app.py:583
+msgid "Open project"
+msgstr "Abrir proyecto"
+
+#: jack_mixer/app.py:631 jack_mixer/app.py:674
+#, python-brace-format
+msgid "Error saving project file '{filename}': {msg}"
+msgstr "Error al guardar el archivo de proyecto '{filename}': {msg}"
+
+#: jack_mixer/app.py:638
+msgid "Save project"
+msgstr "Guardar proyecto"
+
+#: jack_mixer/app.py:691
+msgid "<b>Quit application?</b>"
+msgstr "<b>¿Salir de la aplicación?</b>"
+
+#: jack_mixer/app.py:694
+msgid ""
+"All jack_mixer ports will be closed and connections lost,\n"
+"stopping all sound going through jack_mixer.\n"
+"\n"
+"Are you sure?"
+msgstr ""
+"Todos los puertos de jack_mixer se cerrarán y se perderán las conexiones,\n"
+"deteniendo cualquier audio procesado por jack_mixer.\n"
+"\n"
+"¿Estás segur@?"
```

Comment:
It's used as a way to be more inclusive and avoid using the generic (male) form - "seguro". There are arguments for and against... use the @ and be accused of pandering to the 'woke' crowd; use the generic male form and be accused of helping to perpetuate the patriarchy... I think the best course would be to avoid either altogether and phrase it differently, which is what I've done.

Reply:
Oh, I didn't know that. A gendering problem specific to romance languages. ;) I'll be honest and say that I'm not a fan of neo-graphisms for expressing gender neutrality (and I think that I thought this was an error shows one of the problems with them), but if something can be expressed in a gender neutral way or rephrased, all the better.

Reply:
Yeah, I'm no fan either - I suppose it was just lazy of me. Fixed now though :)